ddl/index.go

```diff
@@ -804,8 +804,13 @@ type addIndexWorker struct {
 
 type reorgIndexTask struct {
 	physicalTableID int64
-	startHandle     kv.Handle
-	endHandle       kv.Handle
+	// indexTableID is used for global index.
+	// Index backfilling on global index will create entries on table but not paritions.
+	// If table is not partitioned, or when backfilling a local index,
+	// indexTableID would be physicalTableID.
+	indexTableID int64
```

Comment:
The worker knows which index it's working on, so `reorgIndexTask` does not need to record extra information?